src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -161,7 +185,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <ProduceReferenceAssembly Condition="'$(ProduceReferenceAssembly)' == ''">false</ProduceReferenceAssembly>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(OutputPath)' == '' ">
+  <PropertyGroup Condition="'$(BaseOutputPathWasSpecified)' != 'true' and '$(_OutputPathWasMissing)' == 'true'">
     <!--
         A blank OutputPath at this point means that the user passed in an invalid Configuration/Platform
         combination.  Whether this is considered an error or a warning depends on the value of
```

Comment:
Can you please update this comment?

Reply:
Sure. Just wanted to get a review from you before I finalize with comments and tests